react/features/prejoin/actions.js

```diff
@@ -554,3 +525,26 @@ function startConference() {
         type: PREJOIN_START_CONFERENCE
     };
 }
+
+/**
+ * Action used to mute/unmute prejoin video.
+ * It destroys/recreates prejoin video tracks.
+ *
+ * @returns {Function}
+ */
+export function mutePrejoinVideo() {
+    return async function(dispatch: Function, getState: Function) {
+        const state = getState();
+        const videoTrack = getVideoTrack(state);
+
+        if (videoTrack === undefined) {
```

Comment:
why is undefined treated differently here?

Reply:
Because there can be a `MUTE_VIDEO` async action dispatched before initialising the prejoin flow (which adds the initially created tracks to the store). That would add the prejoin video twice...